backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -35,6 +37,7 @@
  */
 public class TestEventExport {
   public static final String CSV_API_VERSION = "05Aug2021"; // last time we changed something
+  private static final int FALLBACK_DEFAULT_TEST_LENGTH = 15;
```

Comment:
perhaps include _MINUTES in the name to include the unit.

Reply:
Updated in d84a2fc3